radixdlt/babylon-wallet-android
# Add dApp name to tx status dialog, update UI

## Description
- add dApp name to transaction status updates.
- update font and colors for transaction id row
- split dialog contents into separate composables

## How to test

1.Do successfull/failed tx to verify they are in line with zeplin.
2. Verify that tx proposed by dApp display dapp name on success screen
